openHarmony/openHarmony_node.js

```diff
@@ -3375,9 +3375,13 @@ $.oGroupNode.prototype.importQT = function( path, importSound, extendScene, alig
   // setup the node
   _movieNode.can_animate = false;
   _movieNode.alignment_rule = alignment;
+  
+  // create the temp folder
+  var _tempFolder = new oFolder(this.$.scn.tempFolder.path + "/movImport/" + _element.id);
```

Comment:
Can you add `this.$.oFolder` instead of simply oFolder? This is to ensure it works also if the global scope is unavailable (for example when called from a widget)